rand_core/src/lib.rs

```diff
@@ -68,8 +63,8 @@ pub mod le;
 /// 
 /// This trait encapsulates the low-level functionality common to all
 /// generators, and is the "back end", to be implemented by generators.
-/// End users should normally use [`Rng`] from the [rand] crate, which is
-/// automatically implemented for every type implementing `RngCore`.
+/// End users should normally use `Rng` trait from the [`rand`] crate,
```

Comment:
Yeah, there is no articles in Russian (my native language), so it's a constant problem for me. :) Though this error was preset before my edits, so I think it will be worth to check all crates for such errors.

Reply:
Actually, no it wasn't. :) English is confusing perhaps. E.g.

- the man
- Adam
- the man called Adam
- the man Adam

Equivalently:

- Rng
- the trait Rng

except now we use 'Rng' like an adjective in 'the Rng trait' (compare 'the big man'); at this point I'm less sure of my own language (it may be an error, but if so it's a widespread and persistent one).